package.json

```diff
@@ -3,14 +3,20 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@chakra-ui/icons": "^1.0.6",
+    "@chakra-ui/react": "^1.3.4",
+    "@emotion/react": "^11.1.5",
+    "@emotion/styled": "^11.1.5",
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.5",
     "@testing-library/user-event": "^12.7.0",
     "@types/jest": "^26.0.20",
-    "@types/node": "^14.14.32",
+    "@types/node": "^14.14.34",
     "@types/react": "^17.0.3",
-    "@types/react-dom": "^17.0.1",
+    "@types/react-dom": "^17.0.2",
     "axios": "^0.21.1",
+    "emotion-theming": "^11.0.0",
+    "framer-motion": "^3.10.4",
```

Comment:
Is framer motion necessary for Chakra? I like this library, but we didn't decide about using it just yet.

Reply:
We don't need this : framer-motion . It was in the implementation command : "npm i @chakra-ui/react @emotion/react @emotion/styled framer-motion". I will remove it

Reply:
Summoning @mfurmanit 
What's the definitive answer? 😅 
Maciej showed me that you get errors when you use Chakra components without framer.
![image](https://user-images.githubusercontent.com/40466173/111662249-0df5d100-8810-11eb-9d77-81f142d33e93.png)

Reply:
Correct. If you uninstall `framer-motion` and reinstall `node_modules` you will get the indicated errors.

Reply:
Alright. [Looks like Chakra from v1 forwards requires framer motion. ](https://chakra-ui.com/docs/migration#1-update-your-dependencies)
@kacpergadula leave it then 👌.

Reply:
Sure